crates/neon-runtime/src/napi/bindings/functions.rs

```diff
@@ -19,6 +19,10 @@ mod napi1 {
 
         fn get_value_double(env: Env, value: Value, result: *mut f64) -> Status;
 
+        fn create_date(env: Env, value: f64, result: *mut Value) -> Status;
```

Comment:
This API was added in N-API 5 and not N-API 1. Please add a `napi5` module add add it to loading in `fn load` at the bottom of this file.